src/main/java/io/vertx/kafka/admin/AdminUtilsWrapper.java

```diff
@@ -0,0 +1,116 @@
+package io.vertx.kafka.admin;
+
+import java.util.Map;
+import java.util.Properties;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.kafka.admin.impl.AdminUtilsWrapperImpl;
+import kafka.admin.RackAwareMode;
+import kafka.utils.ZkUtils;
+
+/**
+ * Provides a wrapper around important methods in Kafka's AdminUtils, namely
+ * @see kafka.admin.AdminUtils#createTopic(ZkUtils, String, int, int, Properties, RackAwareMode),
+ * @see kafka.admin.AdminUtils#deleteTopic(ZkUtils, String),
+ * @see kafka.admin.AdminUtils#changeTopicConfig(ZkUtils, String, Properties), and
+ * @see kafka.admin.AdminUtils#topicExists(ZkUtils, String)
+ */
+@VertxGen
+public interface AdminUtilsWrapper {
```

Comment:
I agree with @doernemt even because we do same thing with KafkaConsumer and KafkaProducer for example.